src/ripple/rpc/impl/ShardArchiveHandler.cpp

```diff
@@ -37,11 +37,12 @@ using namespace std::chrono_literals;
 boost::filesystem::path
 ShardArchiveHandler::getDownloadDirectory(Config const& config)
 {
-    return get(config.section(ConfigSection::shardDatabase()),
-               "download_path",
+    return boost::filesystem::path{
                get(config.section(ConfigSection::shardDatabase()),
-                   "path",
-                   "")) /
+                   "download_path",
+                   get(config.section(ConfigSection::shardDatabase()),
+                       "path",
+                       ""))} /
```

Comment:
I don't understand why this changed. What's going on here?

Reply:
For reasons I have since forgotten, my changes somehow changed the requirements for the conversion of the result of `get` to `boost::filesystem::path` from implicit to explicit.  I recall thinking at the time:  Wow, that was fragile code!

This change simply wraps the return of `get` with `boost::filesystem::path{...}`.

Reply:
We can mark this resolved. Thanks!